apps/api/src/domain/models/user-management/utilities/coscrad-date.entity.ts

```diff
@@ -0,0 +1,66 @@
+import { ExternalEnum, PositiveInteger, Year } from '@coscrad/data-types';
+import { DTO } from '../../../../types/DTO';
+import BaseDomainModel from '../../BaseDomainModel';
+
+export enum Month {
+    January = '01',
+    February = '02',
+    March = '03',
+    April = '04',
+    May = '05',
+    June = '06',
+    July = '07',
+    August = '08',
+    September = '09',
+    October = '10',
+    November = '11',
+    December = '12',
+}
+
+export class CoscradDate extends BaseDomainModel {
+    @PositiveInteger({
+        label: 'day of month',
+        description: 'Day of Month',
+    })
+    readonly day: number;
+
+    @ExternalEnum(
```

Comment:
I'm still not happy with my API for this decorator. But let's fight that battle another day.